Header header
float64 closest_point
float64 farthest_point